package.json

```diff
@@ -22,22 +22,22 @@
   },
   "bugs:": "https://github.com/stripe/stripe-node/issues",
   "engines": {
-    "node": "6 || >=8.1"
+    "node": "^6 || ^8.1 || >=10.*"
```

Comment:
Is this correct? OOC, why are we excluding 8.0, is there a known issue with it?

Reply:
looks correct to me, and travis seems to agree. 

yes, paul found an issue with it 😄